torch/onnx/_internal/fx/analysis/unsupported_nodes.py

```diff
@@ -0,0 +1,80 @@
+from __future__ import annotations
+
+import dataclasses
+from typing import Dict, List, Set
+
+from torch.onnx._internal.fx import _pass, diagnostics, function_dispatcher
+
+
+@dataclasses.dataclass
+class UnsupportedFxNodesAnalysisResult(_pass.AnalysisResult):
+    unsupported_op_to_target_mapping: Dict[str, Set[str]]
+
+
+class UnsupportedFxNodesAnalysis(_pass.Analysis):
```

Comment:
Is this already in use?

Reply:
It is. It is invoked before calling `export_fx_to_onnxscript`. It is also how the benchmark gathers all missing op info for a model: by analyzing the diagnostics emitted by this analysis.

Reply:
I see